.circleci/setup_env.sh

```diff
@@ -2,7 +2,7 @@
 
 setup_environment() {
   echo "Setting up env variables"
-  mu_plugins_url="https://github.com/Automattic/vip-mu-plugins-public"
+  mu_plugins_url="https://github.com/Automattic/vip-go-mu-plugins.git"
```

Comment:
Keep the public URL as this may not work when submodules are involved in the repo and when ssh key is not added in github. @vaishaliagola27

Reply:
okay